perma_web/perma/settings/deployments/settings_testing.py

```diff
@@ -8,6 +9,9 @@
     PROJECT_ROOT,
 )
 
+# Default to local functional tests
+USE_SAUCE = os.environ.get('USE_SAUCE', 'False') == 'True'
```

Comment:
Note that any setting can be overridden by an env variable already, via something like DJANGO__USE_SAUCE=True. (This is enabled by https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/settings/utils/environmental_settings.py)

Are the ergonomics worth special-casing this?